list/list_test.go

```diff
@@ -0,0 +1,60 @@
+package list
+
+import (
+	"fmt"
+	"io"
+	"testing"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/stretchr/testify/assert"
```

Comment:
Thank you so much for adding tests, though I think I'd like to avoid adding the `testify/assert` library.

We can pretty much accomplish the same thing with normal `if` statements since `statusView` is simply a string.

Before:
```go
assert.Contains(list.statusView(), "1 item")
```

After:
```go
if !strings.Contains(list.statusView(), "1 item") {
    t.Fatalf("Error: expected view to contain %s", "1 item")
}
```

Reply:
I'll remove it for now. But I think that as we add more test cases, it's pretty much hard to maintain with a normal `if`.

Reply:
Much appreciated ❤️